paper-input-behavior.html

```diff
@@ -335,6 +346,50 @@
     },
 
     /**
+     * Forward focus to inputElement
+     */
+    _onFocus: function() {
+      if (!this._shiftTabPressed) {
+        // iron-autogrow-textarea
+        if (this.inputElement.textarea) {
```

Comment:
Maybe we should make a protected accessor to grab the native element so that you don't have to do this conditional in multiple places?